metrics-config.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<jenkins.metrics.api.MetricsAccessKey_-DescriptorImpl plugin="metrics@3.1.2.11">
+  <!-- TODO: check with Stephen if forcing "evergreen" as key is a bad bad idea
+  FIXME: seems like the "origins" field cannot be saved using the config UI, bug?
+  -->
+  <accessKeys>
+    <jenkins.metrics.api.MetricsAccessKey>
```

Comment:
Are these settings not configurable via the [Configuration as Code](https://github.com/jenkinsci/configuration-as-code-plugin) plugin? 

If not, I believe a ticket should be filed for the Metrics plugin, or for the Configuration as Code plugin, to add support.

Reply:
Agreed.

Reply:
Filed https://github.com/jenkinsci/configuration-as-code-plugin/issues/169

Reply:
Yeah, the latter, please add a reference to the tickets as a comment in the XML when you move it to `configuration/`

Reply:
Done @rtyler